auth/TESLA/Sender_Notary.m

```diff
@@ -0,0 +1,111 @@
+classdef Sender_Notary < handle
+    % SENDER_NOTARY Summary of this class goes here
+    %   Detailed explanation goes here
+
+    properties (Access = public)
+        key_chain_length = 100
```

Comment:
set this to our spec. Create a Testing version of your classes via inheritance (with the small numbers for convenience), and test those.